core-services/ipi-deprovision/clusterpool-pruner.sh

```
@@ -32,11 +32,20 @@ if ! [[ -s /tmp/clusterpools.configured ]]; then
   exit 1
 fi
 
-echo "Discovering configured ClusterImageSets..."
-find $CLUSTERS_PATH/hosted-mgmt/hive/pools -type f -name '*.yaml' | xargs grep 'kind: ClusterImageSet' | awk -F: '{print $1}' | xargs /tmp/yq -N e '.metadata.name' | sort > /tmp/imgsets.configured
+echo "Discovering configured ClusterImageSet names (ClusterImageSet manifests + ClusterPool imageSetRef in git)..."
+find $CLUSTERS_PATH/hosted-mgmt/hive/pools -type f -name '*.yaml' | xargs grep 'kind: ClusterImageSet' | awk -F: '{print $1}' | xargs /tmp/yq -N e '.metadata.name' > /tmp/imgsets.from_manifests
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)